test/napi/lib/workers.js

```diff
@@ -105,7 +105,7 @@ describe("Worker / Root Tagging Tests", () => {
   describe("Multi-Threaded", () => {
     it("should fail to use `get_and_replace`", (cb) => {
       const worker = new Worker(__filename);
-      after(() => worker.terminate());
```

Comment:
These are not ever completing and I'm not sure why. `unref()` accomplishes the same thing and side steps the issue.

https://github.com/neon-bindings/neon/issues/1128